data/stories.yml

```diff
@@ -1,41 +1,792 @@
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_15
+  #     - action: utter_kids_15
+  # - story: kids_16
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_16
+  #     - action: utter_kids_16
+  # - story: kids_17
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_17
+  #     - action: utter_kids_17
+  # - story: kids_18
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_18
+  #     - action: utter_kids_18
+  # - story: kids_19
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_19
+  #     - action: utter_kids_19
+  # - story: kids_20
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_20
+  #     - action: utter_kids_20
+  # - story: kids_21
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_21
+  #     - action: utter_kids_21
+  # - story: kids_22
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_22
+  #     - action: utter_kids_22
+  # - story: kids_23
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: kids_23
+  #     - action: utter_kids_23
+  # - story: spa_1
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: spa_1
+  #     - action: utter_spa_1
+  # - story: spa_2
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: spa_2
+  #     - action: utter_spa_2
+  # - story: pool_1
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: pool_1
+  #     - action: utter_pool_1
+  # - story: pool_2
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: pool_2
+  #     - action: utter_pool_2
+  # - story: pool_3
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: pool_3
+  #     - action: utter_pool_3
+  # - story: retail_1
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: retail_1
+  #     - action: utter_retail_1
+  # - story: retail_2
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: retail_2
+  #     - action: utter_retail_2
+  # - story: retail_3
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: retail_3
+  #     - action: utter_retail_3
+  # - story: retail_4
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: retail_4
+  #     - action: utter_retail_4
+  # - story: retail_5
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: retail_5
+  #     - action: utter_retail_5
+  # - story: retail_6
+  #   steps:
+  #     - intent: greet
+  #     - action: utter_greet
+  #     - intent: retail_6
+  #     - action: utter_retail_6
```

Comment:
If this didn't work well, can you please just delete it? So if other people check out the project, they won't be confused by it.

Reply:
Yes sorry missed deleting these out. Done.